sdk/handleError.js

```diff
@@ -0,0 +1,40 @@
+const get = require('lodash.get');
+const { inspect } = require('util');
+const rp = require('request-promise-native');
+
+const logger = require('../lib/logger');
+
+const { translateErrorCode } = require('binaris-pickle');
+
+class APIError extends Error {}
+
+async function callAPI(options, type = 'post') {
+  const response = await rp[type]({
+    json: true,
```

Comment:
json true appears both here and in callers to `callAPI`. This needs some cleanup.
The only options I see being used are url, body, and method. Method already has its special argument (type). Wouldn't passing all of these explicitly be more readable?
Currently, if the caller sets `simple: true` or `resolveWithFullResponse: true` or even `method: 'GET'` it can get very confusing

Reply:
The design was based on this conversation in the CLI channel in slack https://binarisio.slack.com/archives/C724CG1J5/p1534441584000100.

I agree we should definitely make it clear what is being done automatically and what needs to be passed explicitly. That being said if a user wanted JSON to be false couldn't he simply override that himself?